packages/record/templates/webcam-feed.mustache

```diff
@@ -7,4 +7,4 @@
   height="{{height}}"
   style="display: inline-block"
 ></video>
-<img id="record-icon" src="{{{record_icon}}}">
\ No newline at end of file
+<img id="record-icon" src="{{&record_icon}}" />
```

Comment:
I wasn't familiar with this `{{&` mustache/handlebars syntax and had to look it up. Looks like it's used to return raw values? Is it also needed for the `play_icon` in the `playback-feed.mustache` template? Does `{{{` do the same thing?